_posts/2018-01-24-release-3.2.0.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "Release: 3.2.0"
+date:   2018-01-24
+---
+
+We are pleased to announce the release of OMERO.figure 3.2.0.
+
+New features in OMERO.figure 3.2.0:
+
+ - Page layout supports cropping of the page to fit around image panels ([PR #252](https://github.com/ome/omero-figure/pull/252)).
+ - Creation of new labels from Tags on the image ([PR #254](https://github.com/ome/omero-figure/pull/254)).
+ - Allow renaming of saved figures ([PR #251](https://github.com/ome/omero-figure/pull/251)).
+
```

Comment:
* Rename of Invert -> reverse

Reply:
Is https://github.com/ome/omero-figure/pull/238 a feature or a bugfix?

Reply:
It should go under feature